includes/discovery/sensors/count/fortigate.inc.php

```diff
@@ -21,6 +25,11 @@
 if (! empty($licenseOids)) {
     foreach ($licenseOids as $index => $entry) {
         if (isset($entry['fgLicContractExpiry'])) {
+            // Get current value (for issue #16544)
+            $expirationRaw = snmp_get($device, '.1.3.6.1.4.1.12356.101.4.6.3.1.2.1.2.' . $index, '-OUqnv', 'FORTINET-FORTIGATE-MIB');
```

Comment:
We already have this OID is fgLicContractExpiry, is it not in the format you want? If not try adjusting the snmp walk above it to get the output you need without breaking anything.